artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.Infrastructure/ExternalServices/OpenAITaskGenerationService.cs

```diff
@@ -0,0 +1,80 @@
+using Artificial.Scrum.Master.TaskGeneration.Features.GenerateTasks;
+using Artificial.Scrum.Master.TaskGeneration.Features.Infrastructure;
+using Microsoft.Extensions.Caching.Memory;
+using OpenAI.Interfaces;
+using OpenAI.ObjectModels;
+using OpenAI.ObjectModels.RequestModels;
+using System.Text.Json;
+
+namespace Artificial.Scrum.Master.Infrastructure.ExternalServices;
+internal class OpenAITaskGenerationService : ITaskGenerationService
+{
+    private readonly IMemoryCache _memoryCache;
+    private readonly IOpenAIService _openAIService;
+
+    private readonly TimeSpan CacheTTL = TimeSpan.FromMinutes(45);
+    private const int MaxTokens = 1000;
+
+    public OpenAITaskGenerationService(
+               IMemoryCache memoryCache,
```

Comment:
jak ci się chce to możesz zmienić, copilot nie umie w generowanie konstruktorów bez rozwalania tabów